osgp/platform/osgp-core/src/main/resources/db/migration/V202106630130002351__Add_unique_device_firmware_module_constraint.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE ONLY device_firmware_module
+    ADD CONSTRAINT device_firmware_module_ix_device_id_firmware_module_id UNIQUE (device_id, firmware_module_id);
```

Comment:
I don't think this constraint is needed, as the primary key of this table is already made up of these 2 columns.

Reply:
You are totally right, i missed this, i thought the database was restricted. I reverted this flyway script